api4/webhook_test.go

```diff
@@ -694,29 +662,13 @@ func TestUpdateIncomingHook(t *testing.T) {
 
 		updatedHook, resp := th.SystemAdminClient.UpdateIncomingWebhook(createdHook)
 		CheckNoError(t, resp)
-		if updatedHook != nil {
-			if updatedHook.DisplayName != "hook2" {
-				t.Fatal("Hook name is not updated")
-			}
-
-			if updatedHook.Description != "description" {
-				t.Fatal("Hook description is not updated")
-			}
-
-			if updatedHook.ChannelId != th.BasicChannel2.Id {
-				t.Fatal("Hook channel is not updated")
-			}
-
-			if updatedHook.Username != "" {
-				t.Fatal("Hook username was incorrectly updated")
-			}
-
-			if updatedHook.IconURL != "" {
-				t.Fatal("Hook icon was incorrectly updated")
-			}
-		} else {
-			t.Fatal("should not be nil")
-		}
+
+		require.NotNil(t, updatedHook, "should not be nil")
+		require.Exactly(t, "hook2", updatedHook.DisplayName, "Hook name is not updated")
+		require.Exactly(t, "description", updatedHook.Description, "Hook description is not updated")
+		require.Equal(t, updatedHook.ChannelId, th.BasicChannel2.Id, "Hook channel is not updated")
```

Comment:
this should be reversed, first expected value, then real one.
```suggestion
		require.Equal(t, th.BasicChannel2.Id, updatedHook.ChannelId, "Hook channel is not updated")
```